sova/core/steps/spec.py

```
@@ -361,57 +361,59 @@ async def execute(self, ctx: ExecutionContext) -> StepResult:
-                            args={"issue": ctx.issue_number, "role": "developer"},
-                            auto_execute=True,
-                        ),
-                    ],
-                    notification_message=f"Spec auto-approved for #{ctx.issue_number}, starting developer",
-                    notification_subtitle=f"Researcher finished #{ctx.issue_number}",
-                    result_summary=f"Spec auto-approved (complexity: {spec_complexity}), handed off to developer",
-                    cost_usd=result.cost_usd,
+            updated = re.sub(r"\*\*Status\*\*:\s*\w+", "**Status**: approved", text, count=1)
+            try:
+                spec_path.write_text(updated)
+            except IOError as exc:
```

Ruff (0.16.1)

[warning] Replace aliased errors with `OSError`

Replace `IOError` with builtin `OSError`

(UP024)